src/test/data/entity/whenDidYouPay.ts

```diff
@@ -0,0 +1 @@
+export const whenDidYouPay = { paidDate: '2017-01-01', explanation: 'I paid cash' }
```

Comment:
As we discussed please move it to `src/test/data/entity/responseData.ts`. I do not see a reason for this to leave on its own, it will not be used anywhere else outside response.